sources/Outputs/include/ParCommon.h

```diff
@@ -87,7 +87,7 @@ inline boost::shared_ptr<parameters::MacroParameterSet> buildMacroParameterSetVR
       boost::shared_ptr<parameters::MacroParameterSet>(new parameters::MacroParameterSet(modelType));
   if (modelType == getMacroParameterSetId(constants::remoteVControlParId + "_vr")) {
     macroParameterSet->addParameter(buildParameter("vrremote_Gain", 1.));
-    macroParameterSet->addParameter(buildParameter("vrremote_tIntegral", 1.));
+    macroParameterSet->addParameter(buildParameter("vrremote_tIntegral", 0.01));
```

Comment:
In [3395 OS](https://github.com/dynawo/dynawo/pull/3407) it seems that there are nrts for which you kept this parameter equal to 1. and other for which you changed it to 0.01. Here you impose a default value to 0.01. What is the logic behind this choices?

Reply:
Normally I should have updated everywhere, which nrt are you referring to?

Reply:
[GeneratorPVTfo.par](https://github.com/dynawo/dynawo/pull/3407/files#diff-76d8f13206326289ea71ea478911446d5113e85dfcb6981c70bbd0d61f090ed6), [VRRemote.par](https://github.com/dynawo/dynawo/pull/3407/files#diff-ae98dd81de8474a7efc42b4d210135533f2737ed13bdfed08b29dcc5c5fbc667),

Reply:
my bad, let me correct this

Reply:
OK, then this commit is OK, thx